hwilib/cli.py

```diff
@@ -69,6 +74,7 @@ def process_commands(cli_args):
     parser.add_argument('--fingerprint', '-f', help='Specify the device to connect to using the first 4 bytes of the hash160 of the master public key. It will connect to the first device that matches this fingerprint.')
     parser.add_argument('--version', action='version', version='%(prog)s {}'.format(__version__))
     parser.add_argument('--stdin', help='Enter commands and arguments via stdin', action='store_true')
+    parser.add_argument('--interactive', '-i', help='Use some commands interactively', action='store_true')
```

Comment:
maybe add "Required for some commands"

Reply:
or "Currently required for all device configuration commands"

Reply:
Done